docs/src/main/tut/index.md

```diff
@@ -105,7 +104,7 @@ import remotely._, codecs._
 object protocol {
   val definition = Protocol.empty
     .codec[Int]
-    .specify1[Int, Int]("factorial")
+    .specify1("factorial", Field.strict[Int]("n"), Type[Int])
```

Comment:
@timperrett I think the new DSL is a bit unfriendly compared to what we had. Perhaps we should add the convenience syntax above to the core library and document differences. However, this makes little sense if the plan is to introduce some higher level DSL that is even more friendly to use.